lib/admin/base.php

```diff
@@ -60,6 +60,32 @@ public function icons_url() {
   public static function has_pro() {
     return is_plugin_active('vimeography-pro/vimeography-pro.php');
   }
+  
+  
+  /**
+   * Checks if the user has already signed up for the email list
+   * 
+   * @return boolean
+   */
+  public static function has_user_signed_up() {
+    return get_site_option('vimeography_has_user_signed_up');
+  }
+  
+  /**
+   * Specifies width of gallery, based on if the user has signed up
+   * 
+   * @return string
+   */
+  public static function gallery_width() {
```

Comment:
@alexstandiford You may need to call this something else, as each Vimeography theme already uses a {{gallery_width}} to define its max width.